atlasdb-dagger/src/main/java/com/palantir/atlasdb/services/KeyValueServiceModule.java

```diff
@@ -56,7 +57,14 @@ public KeyValueService provideWrappedKeyValueService(@Named("rawKvs") KeyValueSe
         kvs = TracingKeyValueService.create(kvs);
         kvs = AtlasDbMetrics.instrument(KeyValueService.class, kvs);
         kvs = ValidatingQueryRewritingKeyValueService.create(kvs);
-        kvs = SweepStatsKeyValueService.create(kvs, tss);
+
+        SweepConfig sweepConfig = config.atlasDbRuntimeConfig().sweep();
+        kvs = SweepStatsKeyValueService.create(
+                kvs,
+                tss,
+                sweepConfig::writeThreshold,
+                sweepConfig::writeSizeThreshold);
```

Comment:
Didn't have a `Supplier<Config>` here, I guess the whole class gets regenerated when the config changes? Or is something else happening? Have to init one in `TransactionManagers` too that does have a `Supplier<Config>`.

Reply:
This is just used for the CLIs, console and ETE tests, so we're not worried with having live-reloadable configs here.